src/components/Orders.js

```diff
@@ -8,10 +9,11 @@ class Orders extends React.Component {
       orderRows: []
     };
   }
+
   async componentDidMount() {
     const response = await fetch("http://localhost:3001/orders");
     const json = await response.json();
-    this.setState({ orderRows: json.map(elem => elem) });
+    this.setState({ orderRows: json.map(elem => elem), checked: false });
```

Comment:
you never use this.state.checked in this component, remove useless state variables